app/views/passengers/index.haml

```diff
@@ -47,9 +46,9 @@
           %td= passenger.email
           %td= passenger.phone
           - unless @filters.include? 'Active'
-            %td.yes= yes_no_image passenger.active?
-          %td.yes= yes_no_image passenger.permanent?
-          %td.yes= passenger.expiration_display
+            %td= yes_no_image passenger.active?
```

Comment:
Lastly, sorting these columns doesn't work because, as far as DataTables is concerned, they are empty cells. These also need a 'data-sort' attribute of some sort.

Reply:
Okay so I want you to know that I know how it looks but DataTables really doesn't like straight up booleans and it needs strings.

Reply:
I think it's actually a result of HAML's handling of booleans. It interprets the boolean to mean "should this data attribute be present in the DOM?" :

```
%div{data: {thing: true}}   -makes -  <div data-thing="">
%div{date: {thing: false}}  -makes-  <div>
```

Which is useful in other situations in this, because you can use `hasAttribute('data-thing')` to easily get your `true`s and `false`s into your Javascript logic.